Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -605,4 +606,27 @@ private function getFinalControlContinuationTokensForOpeningToken($openingTokenK
 
         return array();
     }
+
+    private function fixDeclareStrictTypes(Tokens $tokens)
+    {
+        $declares = $tokens->findGivenKind(T_DECLARE);
+
+        foreach ($declares as $index => $declare) {
+            if ($this->isStrictTypeDeclare($index, $tokens)) {
```

Comment:
then my initial commit - fixer shouldn't care if it is a declaration for strict or sth else. should also work for other - as it is.
then don't understand the need for isStrictTypeDeclare check